src/components/AddItem.js

```diff
@@ -12,14 +11,16 @@ import {
   onSnapshot,
 } from '@firebase/firestore';
 import Footer from './Footer';
+import { validateInput } from './Validation';
 
 const AddItem = () => {
   const [items, setItems] = useState([]);
   const [itemName, setItemName] = useState('');
   const [nextPurchase, setNextPurchase] = useState(7);
-  const [lastPurchase, setLastPurchase] = useState(null);
+  const [lastPurchase] = useState(null);
```

Comment:
useState is only used when a value will change eventually. If `lastPurchase` will not change and if that is why you took out `setLastPurchase`, then you could take out useState and define a normal variable.